consul/resource_consul_acl_token.go

```diff
@@ -24,7 +24,6 @@ func resourceConsulACLToken() *schema.Resource {
 			"policies": {
 				Type:     schema.TypeSet,
 				Optional: true,
-				ForceNew: true,
```

Comment:
Is this intentionally included here? I'm not seeing the intent to change this behavior. Maybe it was test related?

Reply:
This was making it impossible to update the policies associated to a token without changing its Access ID and Secret ID. I don't think this is what was expected, especially since there was no test for this.

Pinging @bhuisgen, as the original author he may have more context regarding this.

Reply:
I'm okay with merging as-is but it'd be good to track down the reason there. I think this makes sense to me though.